sections/articles.liquid

```diff
@@ -63,11 +55,27 @@
   {% comment %} CSS variables end {% endcomment %}
 
   <div
-    class="mosaic__wrapper{% if color_palette != blank %} palette-{{ color_palette }}{% endif %}{% if padding_top != blank or padding_top_mobile != blank %} mosaic__wrapper--padding-top{% endif %}{% if padding_bottom != blank or padding_bottom_mobile != blank %} mosaic__wrapper--padding-bottom{%- endif -%}"
+    class="articles__wrapper palette-{{ color_palette }}{% if padding_top != blank or padding_top_mobile != blank %} articles__wrapper--padding-top{% endif %}{% if padding_bottom != blank or padding_bottom_mobile != blank %} articles__wrapper--padding-bottom{%- endif -%}"
     style="{{- variables | escape -}}"
   >
-    <div class="mosaic__container container">
-      <div class="mosaic__list mosaic__list-{{ layout }}{% if blocks.size > 3 %} mosaic__list-two-rows{%- endif -%}">
+    <div class="articles__container container">
+      {%- if title != blank or button_label != blank and button_url != blank -%}
```

Comment:
Perhaps, we can DRY it up like this:
```suggestion
      {% assign show_title = title != blank %}
      {% assign show_button = button_label != blank and button_url != blank  %}
      {%- if show_title or show_button -%}
```

and later:
`{% if show_title %}`
...